pkg/syncer/spec/mutators/deployment.go

```diff
@@ -47,11 +48,12 @@ func (dm *DeploymentMutator) GVR() schema.GroupVersionResource {
 	}
 }
 
-func NewDeploymentMutator(upstreamURL *url.URL, secretLister ListSecretFunc, dnsIP string) *DeploymentMutator {
+func NewDeploymentMutator(upstreamURL *url.URL, secretLister ListSecretFunc, downstreamLogicalName logicalcluster.Name, dnsIP string) *DeploymentMutator {
```

Comment:
> downstreamLogicalName logicalcluster.Name

I'd rather call it `syncTargetLogicalClusterName`. There are no logical clusters or workspaces **downstream** (== in the physical cluster).

Reply:
done